rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/release/Sw360ReleaseService.java

```diff
@@ -1419,4 +1422,38 @@ private void addLicenseInfoResultToJsonSerializableLicensesList(LicenseInfoParsi
                 throw new IllegalArgumentException("Unknown LicenseInfoRequestStatus: " + licenseInfoResult.getStatus());
         }
     }
+
+    public RequestStatus mergeRelease(String mergeTargetId, String mergeSourceId, ReleaseMergeSelector releaseSelection,
+            User sw360User) throws TException {
+
+        validateReleaseMergeSelection(releaseSelection);
+        ComponentService.Iface sw360ComponentClient = getThriftComponentClient();
+        RequestStatus requestStatus = sw360ComponentClient.mergeReleases(mergeTargetId, mergeSourceId, releaseSelection,
+                sw360User);
+
+        if (requestStatus == RequestStatus.IN_USE) {
+            throw new BadRequestClientException("Release already in use.");
+        } else if (requestStatus == RequestStatus.FAILURE) {
+            throw new BadRequestClientException("Cannot merge these releases.");
```

Comment:
Should be an internal server error.
```suggestion
            throw new SW360Exception("Cannot merge these releases.");
```